### DISCUSSION TITLE
🗯 Your feedback needed for an upcoming feature on Discussions!

### DISCUSSION THREAD

**Discussion by jayceeday** [community/community#14721] [Discussions] [answered]
Hello lovely GitHub community! We are working on a new upgrade for Discussions and want your input 🤗

✍️ [**Here is the link** ](https://app.usabilityhub.com/do/97d511e8ce5c/a154)to be the first to see some new mock-ups and give your opinion!

<img width="700" alt="new!!" src="https://user-images.githubusercontent.com/86061938/162771566-8b560d24-9520-4049-a773-a36d39af26cc.png">

**Comment by Andre601** [comment]
Generally speaking, not a fan of displaying the "Most Upvotes Received" and I stated my reasons in the survey.

I do like the preview of the discussions tho.

~~Tho, could you provide a dark-theme image for us dark-theme-users? My poor eyes...~~ (obviously joking)

**Reply by jayceeday** [reply]
Thanks for giving your feedback! It's very helpful.

**Comment by mw491605** [comment]
搞些什么乱七八糟的---- Replied Message ***@***.***>Date04/13/2022 06:59 ***@***.***> ***@***.***>SubjectRe: [github/feedback] 🗯 Your feedback needed for an upcoming feature on Discussions! (Discussion #14721)
Generally speaking, not a fan of displaying the "Most Upvotes Received" and I stated my reasons in the survey.
I do like the preview of the discussions tho.
Tho, could you provide a dark-theme image for us dark-theme-users? My poor eyes... (obviously joking)

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/github/feedback/discussions/14721#discussioncomment-2555572",
"url": "https://github.com/github/feedback/discussions/14721#discussioncomment-2555572",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by jayceeday** [comment]
Still looking for participants if you want to give your input now!

**Comment by suprstarrd** [comment]
For those wondering: @github-community (the public forum organization for GitHub Education) *partially* has these upcoming features (it does not have Most Upvotes Received): https://github.com/orgs/github-community/discussions 

You should look at this *after* you've taken the survey!

**Reply by jayceeday** [reply]
@jbmagination this is now available across GitHub! Organizations can go to their settings, turn on Discussions, then chose a source repository for the organization discussions to come from. 

Here is the [post about the release ](https://github.blog/2022-04-12-whats-new-in-github-discussions-organization-discussions-polls-and-more/) 🎆

**Comment by jayceeday** [comment, marked as answer]
Thank you everyone for your input on this! 🤗 It is greatly appreciated, and baed on your feedback we decided we will not move forward with this Most Upvotes concept. 

Would love to know from you all, is there something else that you would prefer in place of Most helpful/Most upvotes? How would you like to be recognized in Discussions and how would you like to show recognition for others?

**Reply by Andre601** [reply]
For me, maybe a "Most used category" would be an idea?
Alternatively, but this would require GitHub to add this, a "most used tag" list

**Comment by sameue** [comment]
Thought your understanding
